app/components/file_attachment_list/file_attachment_document/index.js

```diff
@@ -16,4 +16,4 @@ function mapDispatchToProps(dispatch) {
     };
 }
 
-export default connect(null, mapDispatchToProps)(FileAttachmentDocument);
+export default connect(null, mapDispatchToProps, null, {forwardRef: true})(FileAttachmentDocument);
```

Comment:
Why are we forwarding the ref here?

Reply:
cause of this https://github.com/mattermost/mattermost-mobile/blob/master/app/components/file_attachment_list/file_attachment.js#L118